File: src/main/java/historywowa/domain/oauth2/application/service/CreateAccessTokenAndRefreshTokenService.java
package historywowa.domain.oauth2.application.service;

import historywowa.domain.oauth2.presentation.dto.res.LoginToken;
import historywowa.domain.member.domain.entity.Role;

public interface CreateAccessTokenAndRefreshTokenService {
    LoginToken createAccessTokenAndRefreshToken(String userId, Role role, String email);
}

File: src/main/kotlin/historywowa/domain/oauth2/application/service/CreateAccessTokenAndRefreshTokenService.kt
package historywowa.domain.oauth2.application.service

import historywowa.domain.member.domain.entity.Role
import historywowa.domain.oauth2.presentation.dto.res.LoginToken

interface CreateAccessTokenAndRefreshTokenService {
    fun createAccessTokenAndRefreshToken(userId: String, role: Role, email: String): LoginToken
}
